slasher/db/schema.go

```diff
@@ -10,6 +10,7 @@ var (
 	historicBlockHeadersBucket        = []byte("historic-block-headers-bucket")
 	indexedAttestationsIndicesBucket  = []byte("indexed-attestations-indices-bucket")
 	validatorsPublicKeysBucket        = []byte("validators-public-keys-bucket")
+	validatorsMinMaxSpanBucket        = []byte("validators-min-max-span-bucket")
```

Comment:
Can we add a comment on what this is used for? Once again, for an inexperienced or new contributor, the name behind this doesn't convey much context and it is not clear what it is needed for.

Reply:
sure. adding